app/controllers/users_controller.rb

```diff
@@ -0,0 +1,22 @@
+class UsersController < ApplicationController
+  before_action :authenticate_user!
+  def index
+    @users = User.all
+  end
+
+  def edit
+    @users = User.find(params[:id])
+  end
+  
+  def destroy
+    @users = User.find(params[:id])
+    @users.destroy
+   
+    redirect_to users_path
+  end
+
+  private
```

Comment:
@apeksha2697  please add one line spacing btw private and methods below

Reply:
done